### PR TITLE
[UI-238] Use clean URLs

### LINKED ISSUE #238: Simplify app pathnames
**What**
Simplify pathnames in the ui documentation app (https://github.com/bufferapp/ui/tree/master/src/documentation/app)

**Why**
The pathnames are currently not very intuitive

**Examples:**
Screenshot 2020-09-29 at 12 19 49
Screenshot 2020-09-29 at 12 20 01

### PR DESCRIPTION
**Pull request by ssvmvss**
## Description

This PR has the changes needed to use clean URLs on the documentation site:
- `BrowserRouter` is used instead of `HashRouter` -> This removes the need of having a `/#/` on our paths. 
- Path on links present on the `Getting started` page were fixed. The current version is redirecting the user to:

image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style and guidelines of this project. 
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the **CHANGELOG** document.
- [ ] I have added tests to cover my changes.
- [ ] I have performed a self-review of my own code
- [x] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] All new and existing tests passed.
- [ ] I have performed the accessibility audit of my UI changes according to the accessibility doc. 
- [ ] [Buffer Engineers] Someone from the Design team reviewed and approved my changes
- [ ] [Buffer Engineers] I have notified the BDS team of my changes in the #proj-design-system Slack channel

### CODE REVIEW

**Comment by ssvmvss** [comment]
Heya @gmzjuliana and @anafilipadealmeida! 👋 🙂 This PR contains only fixes to our documentation site so I'm wondering:
- Do you feel we need to update the `Changelog` for this one? And also:
- Do you feel we need to put another version out there for this or just merge? 🤔 

Would love to know what you think! 🤗

File: src/documentation/app/Home.jsx [hunk 1]
```diff
@@ -89,15 +89,15 @@ function Home() {
           here at Buffer
         </Header>
         <Places>
-          <Place href="/#/Guides/guides">
+          <Place href="/Guides/guides">
```
Comment (gmzjuliana) [inline src/documentation/app/Home.jsx:92]: not blocking but could this be `/guides` without repeating? and so on for the other two

**Review by gmzjuliana — APPROVED** [review]
Looks good to me! thanks for working on this @ssvmvss 🌮  I left a small comment but not blocking. Regarding your question whether to bump the version, I prefer not bumping it and just releasing the next time we merge in a change to the functionality of the lib

File: src/documentation/app/Home.jsx (hunk 1 above)
Comment (anafilipadealmeida) [reply src/documentation/app/Home.jsx:92]: I agree with Juliana, if it's an easy change, could we drop one of them? 🙂

**Review by anafilipadealmeida — APPROVED** [review]
Thank you so much for this @ssvmvss 🙂 I'd agree there's no need for bumping versions or changing the changelog, just merging it sounds fine!

**Comment by ssvmvss** [comment]
Closing this one since we've decided to move all UI library documentation to Notion / Storybook. Improving the current pathnames does not make sense anymore.

**Pull request closed without merging**